Resources/Prototypes/Entities/Structures/Storage/morgue.yml

```
@@ -38,7 +38,7 @@
   - type: EntityStorage
     isCollidableWhenOpen: true
     showContents: false
-    capacity: 1
+    capacity: 999
```

For this it's probably better to have a limit even if very high so we don't get ugly surprises with someone putting 1000 glass shards in it
30 is around the same as crates
```suggestion
    capacity: 30
```